tabpy-server/tabpy_server/app/util.py

```diff
@@ -36,8 +36,8 @@ def validate_cert(cert_file_path):
                           not_before), RuntimeError)
     if now > not_after:
         log_and_raise(https_error +
-                      f'The certificate provided expired on {not_after}.',
-                      RuntimeError)
+                      'The certificate provided expired on {}.'.format(
```

Comment:
Why is this change necessary?

Reply:
you get a syntax error when the code is run

Reply:
only in the sense that the logging syntax for expired now matches the syntax for not_before